homeassistant/components/cloudflare/config_flow.py

```diff
@@ -173,7 +173,7 @@ async def async_step_zone(self, user_input: dict | None = None):
 
     async def async_step_records(self, user_input: dict | None = None):
         """Handle the picking the zone records."""
-        errors = {}
+        errors: dict = {}
```

Comment:
This is not used. should be removed instead of adding type.
And then also remove line 186